vnext/Shared/RuntimeOptions.cpp

```diff
@@ -8,38 +8,104 @@ using std::mutex;
 using std::string;
 
 namespace {
-std::unordered_map<string, int32_t> g_runtimeOptions;
+std::unordered_map<string, int32_t> g_runtimeOptionInts;
+std::unordered_map<string, string> g_runtimeOptionStrings;
 mutex g_runtimeOptionsMutex;
 } // namespace
 
 namespace Microsoft::React {
 
 void __cdecl SetRuntimeOptionBool(string &&name, bool value) noexcept {
```

Comment:
>SetRuntimeOptionBool [](http://example.com/codeflow?start=13&length=20)

Should we avoid duplication and call new functions from the existing? Otherwise we have a risk to get different behavior like we see for this function: it sets 0 for false, and the new one deletes entry.